lib/ddtrace/contrib/extensions.rb

```diff
@@ -86,6 +87,14 @@ def fetch_integration(name)
             registry[name] ||
               raise(InvalidIntegrationError, "'#{name}' is not a valid integration.")
           end
+
+          def silence_logs?
```

Comment:
I think `verbose?` seems like more straightforward name for this option (defaulting to true, and set to false when patching-all).